### DISCUSSION TITLE
Bug : Mobile View - Side Menu Background Transparency Issue

### DISCUSSION THREAD

**Discussion by Harish-M-2003** [pmndrs/zustand#2296] [Bug report]
When accessing the Zustand documentation on mobile devices, an issue arises where the side menu background turns transparent upon opening. This behavior significantly hinders the readability of the content, making navigation and comprehension challenging for users on smaller screens. This problem is not observed on desktop screens, indicating a responsive design issue that needs attention for an optimal user experience.

This issue was initially reported in #2295.

**Comment by dai-shi** [comment]
Thanks for reporting. If you can fix it, it might be helpful!

**Reply by Harish-M-2003** [reply]
Thank you for your prompt attention. I'm glad to see that the issue has been resolved by @CodyJasonBennett
